src/Dialect/ONNX/ONNXOps/OpHelper.hpp.inc

```diff
@@ -31,8 +30,7 @@ inline bool isNoneValue(mlir::Value value) {
 /// Check the defining operation of a value.
 template <typename OP>
 bool definedBy(mlir::Value v) {
-  using namespace mlir;
-  return !v.isa<BlockArgument>() && isa<OP>(v.getDefiningOp());
+  return !v.isa<mlir::BlockArgument>() && isa<OP>(v.getDefiningOp());
```

Comment:
The Windows pipeline is failing as it is more strict about namespacing.

you probaly need `llvm::isa`